src/vm/tieredcompilation.cpp

```diff
@@ -400,6 +408,10 @@ void TieredCompilationManager::ActivateCodeVersion(NativeCodeVersion nativeCodeV
         CodeVersionManager::TableLockHolder lock(pCodeVersionManager);
         ilParent = nativeCodeVersion.GetILCodeVersion();
         hr = ilParent.SetActiveNativeCodeVersion(nativeCodeVersion, FALSE);
+        LOG((LF_TIEREDCOMPILATION, LL_INFO10000, "TieredCompilationManager::ActivateCodeVersion Method=0x%x (%s::%s), code version id=0x%x. SetActiveNativeCodeVersion ret=0x%x\n",
```

Comment:
The format character for the pMethodDesc is incorrect (%x, should be %p and probably formatted as %pM like in the preexisting log call below)